config.json

```diff
@@ -1,7 +1,7 @@
 {
   "language": "Haskell",
   "active": true,
-  "blurb": "",
+  "blurb": "Haskell is a functional programming language which is pure and statically-typed. It's known for lazy evaluation, where evaluation is deferred until necessary, and it's purity, where monads are used for working with side-effects.",
```

Comment:
Seems good. `it's purity` should be `its purity`, right?

I think with that change, that is the only change I have remaining to ask to make.

Reply:
I have corrected this mistake